### PR TITLE
Added Bloodthirsty Modifier

### PR DESCRIPTION
**Pull request by starryburst**

### CODE REVIEW

File: Games/types/Mafia/roles/cards/Bloodthirsty.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                }
+                if (this.data.blood > 0){
+                    if (this.data.blood <= 25){
+                        this.player.queueAlert(`You hunger for blood. You now have ${this.data.blood}% blood left! If you don't kill anybody before the next day, you will die!`);
+                    } else {
+                        this.player.queueAlert(`You hunger for blood. You now have ${this.data.blood}% blood left!`);
+                    }
+                }
+            },
+            "death": function (player, killer, deathType) {
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/Bloodthirsty.js:33]: check death type is not lynch

File: Games/types/Mafia/roles/cards/Bloodthirsty.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+                if (killer === this.player && player !== this.player){
```
Comment (pikulet) [inline Games/types/Mafia/roles/cards/Bloodthirsty.js:34]: max(100, blood + 50)

File: Games/types/Mafia/roles/cards/Bloodthirsty.js (hunk 1 above)
Comment (starryburst) [reply Games/types/Mafia/roles/cards/Bloodthirsty.js:33]: Does the village vote attribute the death cause to a specific person? I thought it was just a general death type that didn't have a specific killer attached to it.

Comment (pikulet) [reply Games/types/Mafia/roles/cards/Bloodthirsty.js:33]: village vote gives to the first person who voted

**Review by pikulet — APPROVED** [review]

**Review by pikulet — APPROVED** [review]

**Pull request merged**